model/Lite/SPDX-Lite.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SPDX-Lite
+
+## Summary
+
+Everything having to do with SPDX-Lite.
+
+## Description
+
+The SPDX-Lite namespace defines concepts related to SPDX-Lite.
```

Comment:
Same comment as above.

Reply:
Thank you @rnjudge -san, I and @NorioKobota -san will make updated revision next week (E/Sep. '23 atmost). And will apply to it your comments in this PR.